.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+// It has ts files setup not applied for the whole project just for any js configuration files at the root
+// such as this file for instance.
+module.exports = {
+  extends: ['eslint:recommended', 'prettier'],
+  plugins: ['prettier'],
+  env: {
+    node: true,
+    mocha: true,
```

Comment:
I greatly prefer to put a `.eslintrc.js` in the `spec` directory with `mocha` there since that's only allowed there and no where else.

Reply:
I have a really bad experience from other projects where it was being used hidden configuration files in subdirectories. For anyone who didn't create that file it is really hard to know they exist. Since eslint uses hidden configuration files I rather keep all configurations in the same place so it is crystal clear all the possible configurations. I could use overrides though if you really just want mocha globals enabled for testing files.